ocrd_utils/ocrd_utils/os.py

```diff
@@ -63,6 +63,20 @@ def unzip_file_to_dir(path_to_zip, output_directory):
     z.extractall(output_directory)
     z.close()
 
+_ocrd_tool_cache = {}
```

Comment:
Good point, I'll do that. With the current command-line-based approach, caching won't help much but with persistent state in networked components, it might help later on.

Reply:
Indeed. And a file-based alternative would only create more problems (concurrent synchronization, lookup slower than in-memory).

Reply:
47198679